src/elmish-debugger/debugger.fs

```diff
@@ -5,30 +5,47 @@ open Fable.Core
 
 [<RequireQualifiedAccess>]
 module Debugger =
+    open FSharp.Reflection
+
+    let inline private duName (x:'a) = 
+        match FSharpValue.GetUnionFields(x, typeof<'a>) with
+        | case, _ -> case.Name
+
+    let [<PassGenericsAttribute>] inline private getCase cmd : obj =
```

Comment:
Oops, I guess I don't fully understand `PassGenericsAttribute` yet. What makes its need go away once inlined? I had assumed it was needed anytime there was a `typeof`. Changing now.